service/src/test/java/uk/gov/hmcts/reform/fpl/config/GeneralInboxLookupConfigurationTest.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.fpl.config;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class GeneralInboxLookupConfigurationTest {
+    private static final String GENERAL_INBOX = "FamilyPublicLaw+generalInbox@gmail.com";
```

Comment:
Why the concatenation?

Reply:
keeping concatenation, changed to `FamilyPublicLaw+PublicLawEmail@gmail.com`